versions.mk

```diff
@@ -24,12 +24,6 @@ vVERSION := v$(VERSION:v%=%)
 GOLANG_VERSION ?= 1.22.1
```

Comment:
Not for this PR, but we should bump this, as @klueska mentioned to be testing 1.22.4 on slack